content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Mengatur Batas Minimum dan Maksimum Memori pada sebuah Namespace
+content_type: task
+weight: 30
+---
+
+
+<!-- overview -->
+
+Laman ini menunjukkan cara untuk mengatur nilai minimum dan maksimum memori yang digunakan oleh Kontainer
```

Comment:
Keep the API name "Container"
```suggestion
Laman ini menunjukkan cara untuk mengatur nilai minimum dan maksimum memori yang digunakan oleh Container
```

Reply:
All resolved in e323b8c5920d6ae02c050306495e784b8e19ad9f